src/LoginRouter.js

```diff
@@ -198,6 +235,18 @@ function (BaseLoginRouter, PrimaryAuthController, VerifyDuoController, MfaVerify
       });
     },
 
+    enrollWindowsHello: function () {
+      this.render(EnrollWindowsHelloController, {
+        provider: 'FIDO',
+        factorType: 'webauthn',
+        Beacon: FactorBeacon
+      });
+    },
+
+    enrollmentWindowsHello: function () {
```

Comment:
Why not "activateWindowsHello"? I.e. a bit confusing to have enrollWindowsHello and enrollmentWindowsHello as methods.